src/ompl/geometric/planners/rrt/TRRT.h

```diff
@@ -167,6 +167,7 @@ namespace ompl
             void setInitTemperature(double initTemperature)
             {
                 initTemperature_ = initTemperature;
+                temp_ = initTemperature_;
```

Comment:
This shouldn't be necessary, since this is also done by setup() and clear().

Reply:
Yes but what if someone calls setup() then calls solve() then sets the initTemperature and then calls solve() again.